server/mysterium_api.go

```diff
@@ -0,0 +1,122 @@
+package server
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	log "github.com/cihub/seelog"
+	"github.com/mysterium/node/identity"
+	"github.com/mysterium/node/server/dto"
+	dto_discovery "github.com/mysterium/node/service_discovery/dto"
+	"net/url"
+)
+
+const (
+	mysteriumApiLogPrefix = "[Mysterium.api] "
+)
+
+var mysteriumApiUrl string
+
+type mysteriumApiClient struct {
+	http HttpClient
+}
+
+func NewClient() Client {
+	return &mysteriumApiClient{
+		http: NewJsonClient(
```

Comment:
Looks quite complicated `http`, `transport` and `httpTransport`

Reply:
A better naming might help. Will fix that.

Reply:
it became mysteriumApi {
     json : NewJsonClient()
}